frontend/scripts/makefile/mobile.toml

```diff
@@ -33,6 +33,29 @@ script = [
 ]
 script_runner = "@shell"
 
+[tasks.appflowy-core-dev-android]
+category = "Build"
+dependencies = ["env_check"]
+run_task = { name = [
+  "setup-crate-type",
+  "sdk-build-android",
+  "post-mobile-android",
+  "restore-crate-type",
+] }
+
+[tasks.sdk-build-android]
+private = true
+script = [
+  """
+    cd rust-lib/
+    rustup show
+    echo "cargo ndk -t arm64-v8a -o ./jniLibs build --features "${FLUTTER_DESKTOP_FEATURES}""
+    cargo ndk -t arm64-v8a -o ./jniLibs build --features "${FLUTTER_DESKTOP_FEATURES}"
```

Comment:
This line could fail for other builds like `x86` for emulators. The solution would be to force specify the platform-api version:
`cargo ndk -t arm64-v8a -p 24 -o ./jniLibs build --features "${FLUTTER_DESKTOP_FEATURES}`
This way even if someone wants to build the backend for `armeabi-7va` they just have to change the target.

Reply:
Yep. I have included `x86_64` in it.

Reply:
Yep. I have included x86_64 in it.